org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java

```diff
@@ -34,6 +34,12 @@
   private static final int TIMEOUT_WARNING_MS = 20000;
   private static final int TIMEOUT_CLEANUP_BUFFER_MS = 5000;
   private static final String LAMBDA_ID = UUID.randomUUID().toString();
+
+  public LambdaRequestHandler() {
+    // create CachedResources once for the entire container
```

Comment:
Quick nit: Can you add a little more detail here about resource reuse - and perhaps a link to the lambda docs describing it?

Reply:
done!